### ISSUE TITLE
[FR] Reenforce last scheduled action at startup

### ISSUE THREAD

**Issue by erkr** [nielsfaber/scheduler-card#757]
### Checklist

- [X] I checked for similar existing requests (both open and closed) before posting.
- [X] My request is generic, other users may benefit from it too.
- [X] I will participate in further discussion about this feature and can test it (if requested) once it's done.

### Proposal

Hi Niels. 
I recently discovered your great scheduler. So much richer than the offering of the default schedule helpers!
Currently, after a restart, the scheduled will nicely handle the next scheduled event. But what about a missed event while HA was down?
My feature request is to add the option to reenforce the last action for the current active time slot, at startup (for scheme based schedules)

Best Eric

### Additional info

Just a check box in the options tab of a schedule: “re-enforce action at startup”

**Comment by erkr** [comment]
Still hoping ☺️

**Comment by sddgit** [comment]
I’m not sure if it’s your issue, but take a look at https://github.com/nielsfaber/scheduler-component/issues/234. This is a discussion around startup behaviour, and how it was ultimately implemented.

**Comment by erkr** [comment]
> I’m not sure if it’s your issue, but take a look at https://github.com/nielsfaber/scheduler-component/issues/234. This is a discussion around startup behaviour, and how it was ultimately implemented.

Yes, thanks for pointing to that description. In believe it applies. Now I read the intended behaviour, I realised that it probably have to redo my testing as I manually changed the setting and expected it to restore after a restart. That didn't work.
But I will give it a retry where I bring HA down around a schedule trigger and check if it caches up on that one. If that works I'm totally happy. Will update on that!
